### ISSUE TITLE
'Missing keys' warning when customizing supported translation

### ISSUE THREAD

**Issue by BasileNouvellet** [onfido/onfido-sdk-ui#1240]
### What was the expected behaviour?

No `Missing keys` warning in the console when customizing supported translation.

### What happened instead?

We have a `Missing keys` warning.

### Version info:

- Onfido SDK: 6.2.0

### Solution

There is a typo in [the polyglot file](https://github.com/onfido/onfido-sdk-ui/blob/6.2.0/src/locales/polyglot.js#L59).

It should be

```js
findMissingKeys(defaultKeys, customKeys, customLanguageConfig.locale)
```

instead of 

```js
findMissingKeys(defaultKeys, customKeys, customLanguageConfig)
```

**Comment by kopijunkie** [comment]
Hi @BasileNouvellet 

We've just released [6.4.0](https://github.com/onfido/onfido-sdk-ui/releases/tag/6.4.0) that includes the fix for the "Missing keys" warning you have encountered.

**Comment by kopijunkie** [comment]
Will close this ticket now, but feel free to re-open if you find that you are still having issues.
